### ISSUE TITLE
Compare completion date to start date

### ISSUE THREAD

**Issue by troddick** [USGS-WiM/SiGLDMS#69]
Validate that the project completed date came AFTER the start date?
Probably would also apply to site sampling date, too.

**Comment by troddick** [comment]
"You've entered a completion date that preceeds the start date" modal

**Comment by troddick** [comment]
Done

**Comment by jlbruce** [comment]
When creating a project, it throws the error, but then it lets you save the project with the error. Any way we could clear the field upon error, forcing them to re-enter a good value? (You would clear whichever field caused the error - if they enter the completed first, and the error gets thrown on the start date, then the start date clears.)

**Comment by jlbruce** [comment]
I was also testing editing a project with other open errors, and when I went to enter a start date, it gave me the "completion date after" modal/toaster error, even though I hadn't entered a completion date yet for this project (at all). Even after this project's errors had been resolved, it continued to give me that error, and wouldn't save a start date at all. (See project "Test - break for me one more time")

This is now happening with every project - the start date always throws the error. For a while I thought it was tied to the status (if it was expecting a completion date, and didn't have one, it threw the error), but this is happening with projects that have the completion date grayed out too.

Or, maybe not - I switched the status from active - undetermined (initial error), to completion (entered a good completion date), back to active - undetermined (it cleared completion date), and then it let me change the start date without an error.

I'm so confused now!
